src/System.Management.Automation/engine/ExperimentalFeature/ExperimentalFeature.cs

```diff
@@ -114,7 +114,10 @@ static ExperimentalFeature()
                     description: "New parameter set for ForEach-Object to run script blocks in parallel"),
```

Comment:
@adityapatwardhan, your last commit had 15 failures in `PowerShell-CI-macos`
(These are 5 of the failures)

assignment statement parsing.Parse error expected: <<$cricket ?= $soccer>>.Error count
```powershell
Expected 1, but got 2.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/tools/Modules/HelpersLanguage/HelpersLanguage.psm1: line 89
89:         It "Error count" { $errors.Count | Should Be $expectedErrors.Count }
```
assignment statement parsing.Parse error expected: <<$cricket ?= $soccer>>.Error position (iteration:0)
```powershell
Expected 0, but got 10.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/tools/Modules/HelpersLanguage/HelpersLanguage.psm1: line 105
105:             It "Error position (iteration:$i)" -Pending:$SkipAndCheckRuntimeError { $acutalPostion | Should Be $expectedOffsets[$i] }
```
assignment statement parsing.Parse error expected: <<$cricket ?= $soccer>>.Error Id (iteration:1)
```powershell
Expected $null, but got 'InvalidLeftHandSide'.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/tools/Modules/HelpersLanguage/HelpersLanguage.psm1: line 102
102:             It "Error Id (iteration:$i)" { $errorId | Should Be $expectedErrors[$i] }
```
assignment statement parsing.Parse error expected: <<$cricket ?= $soccer>>.Error position (iteration:1)
```powershell
Expected $null, but got 0.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/tools/Modules/HelpersLanguage/HelpersLanguage.psm1: line 105
105:             It "Error position (iteration:$i)" -Pending:$SkipAndCheckRuntimeError { $acutalPostion | Should Be $expectedOffsets[$i] }
```
null coalescing statement parsing.Parse error expected: <<$x??y>>.Error count
```powershell
Expected 1, but got 0.
at <ScriptBlock>, /Users/vsts/agent/2.158.0/work/1/s/test/tools/Modules/HelpersLanguage/HelpersLanguage.psm1: line 89
89:         It "Error count" { $errors.Count | Should Be $expectedErrors.Count }
```